tests/lambda_endpoints/test_queries.py

```diff
@@ -0,0 +1,708 @@
+"""Test queries lambda"""
+import datetime
+import json
+
+import pytest
+from sqlalchemy.orm import Session
+
+from sds_data_manager.lambda_code.SDSCode import queries
+from sds_data_manager.lambda_code.SDSCode.database import models
+
+
+@pytest.fixture()
+def setup_test_data(test_engine):
+    metadata_params = {
+        "file_path": "test/file/path/imap_hit_l0_raw_20251107_20251108_v02-01.pkts",
+        "instrument": "hit",
+        "data_level": "l0",
+        "descriptor": "raw",
+        "start_date": datetime.datetime.strptime("20251107", "%Y%m%d"),
+        "end_date": datetime.datetime.strptime("20251108", "%Y%m%d"),
+        "version": "v02-01",
+        "extension": "pkts",
+    }
+
+    # Add data to the file catalog
+    with Session(test_engine) as session:
+        session.add(models.FileCatalog(**metadata_params))
+        session.commit()
+
+    return ""
```

Comment:
Is this just to avoid the pre-commit warning? Can you remove the return altogether or return None.

Reply:
That's weird. I did have  `yield ""` because I thought ruff would have an issue if it was just `yield`, but it was complaining about it being `yield` instead of `return` and it looks like it might have automatically changed it to `return`. It does look like like it still works as only `return`, so I'll remove the empty string.

Reply:
Nevermind, if I just have `return` or `return None` it looks like either black or ruff removes it from the code and then ruff complains about nothing being returned lol

Reply:
`yield session` might make sense here? Which would be followed by a `session.close()` below I think.
https://stackoverflow.com/questions/74921132/how-to-populate-db-from-fixture-before-test